metricbeat/module/kibana/status/data.go

```diff
@@ -0,0 +1,43 @@
+package status
+
+import (
+	"encoding/json"
+
+	"github.com/elastic/beats/libbeat/common"
+	s "github.com/elastic/beats/metricbeat/schema"
+	c "github.com/elastic/beats/metricbeat/schema/mapstriface"
+)
+
+var (
+	schema = s.Schema{
+		"uuid": c.Str("uuid"),
+		"name": c.Str("name"),
+		"version": c.Dict("version", s.Schema{
+			"number": c.Str("number"),
+		}),
+		"status": c.Dict("status", s.Schema{
+			"overall": c.Dict("overall", s.Schema{
+				"state": c.Str("state"),
+			}),
+		}),
+		"metrics": c.Dict("metrics", s.Schema{
+			"requests": c.Dict("requests", s.Schema{
+				"total":       c.Int("total"),
+				"disconnects": c.Int("disconnects"),
+			}),
+			"concurrent_connections": c.Int("concurrent_connections"),
+		}),
+	}
+)
+
+type OverallMetrics struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type OverallMetrics should have comment or be unexported